src/Components/CustomComponent/FormFields.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { InputText } from 'primereact/inputtext';
+import { FormFieldsProps } from '../../Type/CommonType';
+
+const FormFields: React.FC<FormFieldsProps> = ({ label, value, onChange, type = 'text' }) => {
+  return (
+    <div>
+      <label htmlFor={label} className="block text-sm font-medium text-gray-700">
+        {label}
+      </label>
+      {type === 'file' ? (
+        <input
```

Comment:
using switch condition would be nice , since you can add more type of fields here